drivers/sensor/lm75/lm75.c

```diff
@@ -67,6 +67,13 @@ static int lm75_sample_fetch(const struct device *dev,
 	enum pm_device_state pm_state;
 	int ret;
 
+#ifdef CONFIG_PM_DEVICE_RUNTIME
```

Comment:
I don't think you need `CONFIG_PM_DEVICE_RUNTIME`, as `device_runtime.h` defines this function are no op when `CONFIG_PM_DEVICE_RUNTIME` is not defined.

Reply:
I wasnt sure about this, but I looked at other drivers and it seems that this is done in most of them..
for example: https://github.com/zephyrproject-rtos/zephyr/blob/53c350e17bf09b2165318308e203f5d536f17fe7/drivers/i2c/i2c_ll_stm32.c#L176
should I remove it anyways?

Reply:
there's no need to wrap, functions are noops if runtime is not enabled. Also, the code below (state get) should be removed.

Reply:
ok, yeah the state get code is what prevents the driver from working, I will remove it too.

Reply:
done..